resources/serverless/charts/webhook/templates/deployment.yaml

```diff
@@ -48,6 +48,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: METRICS_DOMAIN
```

Comment:
Add a comment that it is needed because webhook is based on knative

Reply:
Good point. Done